src/Bundler.js

```diff
@@ -98,7 +97,9 @@ class Bundler extends EventEmitter {
         }
       }
     } catch (err) {
-      this.logger.warn(err);
+      setTimeout(() => {
+        logger.warn(err);
+      }, 500);
```

Comment:
Whats the 0.5 second delay for?

Reply:
So that the warning doesn't get removed when the builder actually starts and does logger.clear()

Reply:
Nvm this is no longer necessary in the latest version...